docs/getting-started/04-publish-adapter.md

```diff
@@ -1 +1,82 @@
-# Publishing your adapter
\ No newline at end of file
+# Publishing your adapter
+## one time to do for the release/publishing
+### ioBroker developer portal
+go to https://www.iobroker.dev/adapter-check
+
+Check your adapter by giving your adapter git repository. This will check a lot of important standarts. The following could be ignored:
+
+[E200] "Not found on npm. Please publish" 
+
+[W400] "Cannot find "your adapter" in latest repository
+
+because we have not published. Have a look to all other errors and warnings.
+
```

Comment:
maybe also add a section to look for infos and best practices in https://github.com/ioBroker/ioBroker.repositories#development-and-coding-best-practices and pot also already https://github.com/ioBroker/ioBroker.repositories/blob/master/REVIEW_CHECKLIST.md ... or do we have a "prepare your adapter for repo" oage too? ;-)